packages/contact-center/station-login/src/station-login/station-login.presentational.tsx

```diff
@@ -1,14 +1,143 @@
-import React from 'react';
+import React, {useEffect} from 'react';
+import {StationLoginPresentationalProps} from './station-login.types';
 
-import {IStationLoginProps} from './station-login.types';
+const StationLoginPresentational: React.FunctionComponent<StationLoginPresentationalProps> = (props) => {
+  const {name, teams, loginOptions, login, logout, setDeviceType, setDialNumber, setTeam} = props; // TODO: Use the  loginSuccess, loginFailure, logoutSuccess props returned fromthe API response via helper file to reflect UI changes
+
+  useEffect(() => {
+    const teamsDropdown = document.getElementById('teamsDropdown') as HTMLSelectElement;
```

Comment:
Are we not taking the coderabbitai comment to use `useRef`?

Reply:
This is removable code so I have ignored comments on the presentational component file.